website/docs/compose/setting-up-compose.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Setting up Compose
+description: Podman Desktop can install the Compose engine.
+sidebar_position: 2
+keywords: [compose]
+tags: [compose]
+---
+
+# Setting up Compose
+
+Podman Desktop can install the Compose engine.
+
+#### Procedure
+
+1. Install the Compose engine: in the Podman Desktop status bar, click on **<icon icon="fa-solid fa-download" size="lg" />Compose**, and follow the prompts.
+1. To use the Compose engine from your terminal, [set the DOCKER_HOST variable](/docs/migrating-from-docker/using-the-docker_host-environment-variable).
+
+#### Verification
+
+1. Open a terminal.
+1. The Compose engine is in your PATH, therefore, you can display the Compose engine version:
```

Comment:
```suggestion
1. The Compose reference implementation is in your PATH, therefore, you can display the Compose engine version:
```

Referring to Compose engine is fine in the other parts of the docs, but now we are talking about the CLI, I think we should be saying reference implementation.

compose-spec.io refers to it as a reference implementation as we should try to reference it as much as possible.